dbt/include/bigquery/macros/relations/materialized_view/describe.sql

```diff
@@ -0,0 +1,45 @@
+{% macro bigquery__describe_materialized_view(relation) %}
```

Comment:
still need to get partition information here and make sure all options show up (trying via `INFORMATION_SCHEMA.TABLE_OPTIONS` hit a slight snag as some are options for external tables only.